.github/workflows/build.yaml

```diff
@@ -23,11 +22,11 @@ jobs:
       fail-fast: false
       matrix:
         channel:
-          - master
+          # When updating this version, also update the version in flutter-version.txt.
+          - 2.4.0-4.0.pre
```

Comment:
can you add a comment describing how often we need to update this and which version we should be selecting when we do?

Reply:
done!